### PR TITLE
Windoors and Directional Windows Now Block Atmos

### LINKED ISSUE #7838: Windoors Dont Block Atmos
Windoors Dont Block Atmos

-Enclose a room and have a windoor to access.

-Explode plasma canister in room

-Plasma will leak out through windoor

### LINKED ISSUE #8733: Extra room gas setter
Find on which station and which room, remove the second gas setter object from the scene.

```
Room number: 24 was already added, cant add AirRoomGasSetter (1)
Systems.Atmospherics.AtmosSystem:AddToListRoom (int,Systems.Atmospherics.RoomGasSetter) (at /root/UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosSystem.cs:113)
Systems.Atmospherics.RoomGasSetter:SetUp () (at /root/UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/RoomGasSetter.cs:65)
```

### PR DESCRIPTION
**Pull request by ThatDan123**
CL: [New] Windoors and Directional Windows now block atmos

Fixes #7838
Fixes #8733

### CODE REVIEW

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosSimulation.cs [hunk 1]
```diff
@@ -64,7 +64,10 @@ public void Run()
 		private void Update(MetaDataNode node)
 		{
 			nodes.Clear();
-			nodes.Add(node);
+
+			//Gases are frozen within walls and isolated tiles (closed airlocks) so dont do gas equalising
+			var doEqualise = node.IsOccupied == false && node.IsIsolatedNode == false;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosSimulation.cs:69]: isn't this stuff handled by neighbour calculations? E.G tile is neighbour to a, but not to b  on the other side?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosSimulation.cs:69]: > isn't this stuff handled by neighbour calculations? E.G tile is neighbour to a, but not to b on the other side?

I had to refactor that a while ago for the heat conductivity stuff, so nodes will always have all of their neighbours.

**Comment by ThatDan123** [comment]
Codacy can be ignored for both of those issues, funny how i said it was rare in the other PR and yet its the same situation lol

**Comment by corp-0** [comment]
codacy doesn't complain about methods with ``out`` when they are named ``TrySomethingSomething`` and return a bool. It is to indicate it will out something, like ``TryGetComponent<T>``

**Review by corp-0 — APPROVED** [review]

**Pull request merged**